### ISSUE TITLE
Is the value loss scaled properly for gradient accumulation in value_loss() in verl/workers/utils/losses.py?

### ISSUE THREAD

**Issue by x-zb** [verl-project/verl#7110]
I'm wondering if the value loss is correctly scaled for gradient accumulation in DP, since I cannot find any code doing this. On the contrast, in ppo_loss() and sft_loss(), parameters such as `batch_num_tokens` and `dp_size` are passed to the loss function for scaling.

**Comment by x-zb** [comment]
I realized my local code is the version released in June. I see it is already fixed.
